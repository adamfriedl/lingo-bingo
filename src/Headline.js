// @flow

import React from 'react';
import { Heading } from 'rebass';

const Headline = (props: { text: string }) => (
  <Heading
    style={{
      fontFamily: 'Damion',
      textShadow: '-4px 4px 1px #79e4a6'
    }}
    f={[5, 6, 7]}
    my={[3, 4]}
    center
    color="blue9"
    children={props.text}
  />
);

export default Headline;
