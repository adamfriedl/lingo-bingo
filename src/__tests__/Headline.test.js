import React from 'react';
import { shallow } from 'enzyme';
import Headline from '../Headline';

it('renders', () => {
  const component = shallow(<Headline />);
  expect(component).toMatchSnapshot();
});
