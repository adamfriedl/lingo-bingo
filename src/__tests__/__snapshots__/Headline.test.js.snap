// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Headline />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "center": true,
      "children": undefined,
      "color": "blue9",
      "f": Array [
        5,
        6,
        7,
      ],
      "my": Array [
        3,
        4,
      ],
      "style": Object {
        "fontFamily": "Damion",
        "textShadow": "-4px 4px 1px #79e4a6",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "center": true,
        "children": undefined,
        "color": "blue9",
        "f": Array [
          5,
          6,
          7,
        ],
        "my": Array [
          3,
          4,
        ],
        "style": Object {
          "fontFamily": "Damion",
          "textShadow": "-4px 4px 1px #79e4a6",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
